src/config.js

```diff
@@ -0,0 +1,66 @@
+const chalk = require('chalk');
+const { join } = require('path');
+
+const getServePath = (dir, path) => {
+  if (typeof path !== 'string' || typeof dir !== 'string') {
+    return { path: undefined };
+  }
+
+  const resolvedPath = join(dir, path);
+  if (!resolvedPath.startsWith(dir)) {
```

Comment:
I've added a test case for this:
https://github.com/netlify-labs/netlify-plugin-lighthouse/pull/45/commits/d892f583fec5988434c87ea916bda1f38cf67493#diff-717adf98883419d80e527cbc26159056R114

It is best practice to verify that after the `join` the resulting directory is under the base directory to prevent path traversal.

For example `join("dir", "../")` returns `./`, which will access a directory outside the desired context.